youtube_dl/extractor/piksel.py

```diff
@@ -17,18 +17,6 @@
 class PikselIE(InfoExtractor):
     _VALID_URL = r'https?://player\.piksel\.com/v/(?P<id>[a-z0-9]+)'
     _TESTS = [
-        {
-            'url': 'http://player.piksel.com/v/nv60p12f',
```

Comment:
replace the old test with a test with subtitles.

Reply:
will do.